Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonThemeComboBox.cs

```diff
@@ -12,190 +12,218 @@
-        /// </summary>
-        [Browsable(false)]
-        [DesignerSerializationVisibility(DesignerSerializationVisibility.Hidden)]
-        public new ComboBoxStyle DropDownStyle
-        {
-            get => base.DropDownStyle;
-            set => base.DropDownStyle = value;
-        }
-
-        /// <summary>
-        /// Gets or sets the items in the KryptonComboBox.
-        /// </summary>
-        [Browsable(false)]
-        [DesignerSerializationVisibility(DesignerSerializationVisibility.Hidden)]
-        public new ComboBox.ObjectCollection Items => base.Items;
-
-        /// <summary>Gets or sets the draw mode of the combobox.</summary>
-        /// <value>The draw mode of the combobox.</value>
-        [Browsable(false)]
-        [DesignerSerializationVisibility(DesignerSerializationVisibility.Hidden)]
-        public new DrawMode DrawMode
-        {
-            get => base.DrawMode;
-            set => base.DrawMode = value;
-        }
-
-        /// <summary>
-        /// Gets or sets the StringCollection to use when the AutoCompleteSource property is set to CustomSource.
-        /// </summary>
-        [Browsable(false)]
-        [DesignerSerializationVisibility(DesignerSerializationVisibility.Hidden)]
-        public new AutoCompleteStringCollection AutoCompleteCustomSource
-        {
-            get => base.AutoCompleteCustomSource;
-            set => base.AutoCompleteCustomSource = value;
-        }
-
-        [Browsable(false)]
-        [DesignerSerializationVisibility(DesignerSerializationVisibility.Hidden)]
-        public new int SelectedIndex { get => base.SelectedIndex; set => base.SelectedIndex = value; }
-
-        #endregion
-    }
+		/// <value>The default palette mode.</value>
+		[Category(@"Visuals")]
+		[Description(@"The default palette mode.")]
+		[DefaultValue(PaletteMode.Microsoft365Blue)]
+		public PaletteMode DefaultPalette
+		{
+			get => _defaultPalette;
+
+			set
+			{
+				_defaultPalette = value;
+
+				UpdateDefaultPaletteIndex(value);
+			}
+		}
+
+		/// <summary>
+		/// Gets and sets the ThemeSelectedIndex.
+		/// </summary>
+		[Category(@"Visuals")]
+		[Description(@"Theme Selected Index. (Default = `Office 365 - Blue`)")]
+		[DefaultValue((int)PaletteMode.Microsoft365Blue)]
+		public int ThemeSelectedIndex
+		{
+			get => _selectedIndex = _defaultPaletteIndex ?? 30;
+
+			private set => _selectedIndex = SelectedIndex = value;
+		}
+
+		private void ResetThemeSelectedIndex() => _selectedIndex = _defaultPaletteIndex ?? 30;
+
+		private bool ShouldSerializeThemeSelectedIndex() => _selectedIndex != _defaultPaletteIndex;
+
+		/// <summary>
+		/// Gets and sets the ThemeSelectedIndex.
+		/// </summary>
+		[Category(@"Visuals")]
+		[Description(@"Custom Theme to use when `Custom` is selected")]
+		[DefaultValue(null)]
+		public KryptonCustomPaletteBase? KryptonCustomPalette { get; set; }
+
+		//[EditorBrowsable(EditorBrowsableState.Never)]
+		public KryptonManager Manager
+		{
+			get;
```

Comment:
I think this should return the owner form manager before creating a new one

Reply:
`Manager` is now assignable